pkg/networkgraph/testutils/utils.go

```diff
@@ -27,6 +27,32 @@ func GetDeploymentNetworkEntity(id, name string) *storage.NetworkEntityInfo {
 	}
 }
 
+func GetExtSrcNetworkEntityDiscovered(id, name, cidr string, isDefault bool, clusterID string, isDiscovered bool) *storage.NetworkEntity {
```

Comment:
I think something is wrong here. I don't think this is needed and the implementation might not be corrent.

Reply:
For consistency I think it might be better to add a `isDiscovered` parameter to the existing `GetExtSrcNetworkEntity` function (and by extension `GetExtSrcNetworkEntityInfo`)

_Or_ since that one already produces non-discovered entities, make this implementation always set `Discovered` to `true` and drop the `isDiscovered` parameter altogether

Reply:
Unfortunately there are 108 calls to `GetExtSrcNetworkEntity` and `GetExtSrcNetworkEntityInfo`. Otherwise I would have added the parameter. Probably I could automate adding false to each call to those functions. I don't know why go doesn't allow for optional parameters. I renamed this `GetExtSrcNetworkEntityWithDiscovered` to more accurately reflect that `isDiscovered` can be true or false. I then made is so that ` GetExtSrcNetworkEntityInfo` calls `GetExtSrcNetworkEntityInfoWithDiscovered` so that there is less code duplication. Let me know if you think adding the new `isDiscovered` parameter to the existing functions is better than adding new ones.

Reply:
Done